api/v1beta1/flinkcluster_types.go

```diff
@@ -434,6 +435,18 @@ type JobManagerIngressStatus struct {
 	URLs []string `json:"urls,omitempty"`
 }
 
+// JobManagerServiceStatus defines the observed state of FlinkCluster
+type JobManagerServiceStatus struct {
+	// The name of the Kubernetes jobManager service.
+	Name string `json:"name"`
+
+	// The state of the component.
+	State string `json:"state"`
+
+	// The port of NodePort.
```

Comment:
Could we update it to "(Optional) The node port, present when AccessScope is `NodePort`"?